spec/features/web_intake/returning_filer_spec.rb

```diff
@@ -37,7 +37,7 @@
     click_on I18n.t('general.continue')
 
     # backtaxes
-    check Rails.configuration.gyr_current_tax_year - 2
+    check (Rails.configuration.gyr_current_tax_year - 2).to_s
```

Comment:
This and the below identical changes are because Capybara was throwing a warning that we shouldn't use an int as an argument to `check`, and that anything other than a string or symbol would throw an error in a future version